includes/wikia/services/WikiService.class.php

```diff
@@ -91,8 +91,23 @@ function() use ( $wiki, $useMaster, $excludeBots, $limit ) {
 		);
 
 		$userIds = array_unique( array_merge( $userIds, $adminIds ) );
+		$result = [];
 
-		return $userIds;
+		foreach ($userIds as $admin) {
```

Comment:
Should be spacey per conventions:
```suggestion
		foreach ( $userIds as $admin ) {
```

Reply:
Also, this might all be slightly cleaner as an array filter rather than building a new array, e.g. something like:
```php
		$userIds = array_filter( array_unique( array_merge( $userIds, $adminIds ) ), function ( $userId ) {
			$user = \User::newFromId( $userId );
			$user->load();
			return !$user->isAnon();
		} );
```

Reply:
Uhm. I was thinking about doing that but after I've struggled a bit with `city_founding_user` I've forgot to refactor this part.

Done in baa0afd099c